src/Sarif.Converters/ContrastSecurityConverter.cs

```diff
@@ -117,12 +112,12 @@ internal Result CreateResult(ContrastLogReader.Context context)
             {
                 case ContrastSecurityRuleIds.AntiCachingControlsMissing:
                 {
-                    return ConstructAntiCachingControlsMissingResult(context.Properties);
+                    return ConstructAntiCachingControlsMissingResult(context);
```

Comment:
>context [](start = 69, length = 7)

All of these changes involve passing in the entire context so we can use the `RuleId` it contains, rather than hard-coding the rule id in each handler.